apps/backoffice-v2/src/common/components/molecules/DocumentTracker/interfaces.ts

```diff
@@ -0,0 +1,13 @@
+import { TWorkflowDefinitionById } from '@/domains/workflow-definitions/fetchers';
+import { TWorkflowById } from '@/domains/workflows/fetchers';
+
+export interface IDocumentTrackerProps {
+  workflow: TWorkflowById;
+  plugins: Array<
+    | NonNullable<NonNullable<TWorkflowDefinitionById['extensions']>['apiPlugins']>[number]
+    | NonNullable<
+        NonNullable<TWorkflowDefinitionById['extensions']>['childWorkflowPlugins']
+      >[number]
+    | NonNullable<NonNullable<TWorkflowDefinitionById['extensions']>['commonPlugins']>[number]
+  >;
+}
```

Comment:
These are not needed right? It seems at least workflow is not in use and if yes maybe we can pass just the properties in use and not the full workflow.